gnosis_vpn-lib/src/connection/destination.rs

```diff
@@ -52,6 +52,10 @@ impl Destination {
         metas.sort();
         metas.join(", ")
     }
+
+    pub fn get_meta(&self, key: &str) -> String {
+        self.meta.get(key).unwrap_or(&"<unknown>".to_string()).clone()
```

Comment:
Why would you want to hide the no value case with `"unknown"`?

Reply:
No, just handle it on the display side. (or wherever you want to show that `unknown` value)
The value from this module should strictly be `None` when you query a non existing key.